spooky-doodle/Bangazon-Website
# Accounts

# Description

Added first name, last name and street address to Account Edit page.

Fixes #17 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Login.
Click on your email address to view the Profile page.  
Make some changes and click "Save".
You should see a confirmation message and your new data should persist.



# Checklist:
